routes/todo.go

```diff
@@ -0,0 +1,63 @@
+package routes
+
+import (
+	"errors"
+	"fmt"
+	"github.com/gofiber/fiber/v2"
+	"strconv"
+)
+
+func createErrorMap(err error) fiber.Map {
+	return fiber.Map{
+		"success": false,
+		"error":   err.Error(),
+	}
+}
+func (h handler) GetAll(c *fiber.Ctx) error {
+	todoList := h.DB.FetchAll()
+
+	return c.Status(fiber.StatusOK).JSON(fiber.Map{
+		"success": true,
+		"data":    todoList,
+	})
+
```

Comment:
Please use a formatter before you upload your code.

Reply:
fixed f63125eabb1fea3236004ab3f7bfcd546b99ac49